localstack/services/s3/presigned_url.py

```diff
@@ -790,8 +790,9 @@ def validate_post_policy(
     form_dict = {k.lower(): v for k, v in request_form.items()}
     for condition in conditions:
         if not _verify_condition(condition, form_dict, additional_policy_metadata):
+            str_condition = str(condition).replace("'", '"')
```

Comment:
Q: do you have an example of when this would happen? I think it's not covered by the newly added test?
If it's already casted to str, why do we still need to replace the single quote? Wouldn't we end up with double quotes inside the string?

Reply:
https://github.com/localstack/localstack/pull/10889/commits/bafa871166b7da2fd96088b7ed3494d53a8fa2de should be a bit clearer, sorry for not mentioning it in the PR description! I realized after that I left the util in the previous tests, now it's cleaner 😄